### ISSUE TITLE
package name conflicts with existing project

### ISSUE THREAD

**Issue by rhys-vdw** [Megaputer/dts-css-modules-loader#2]
[Package name is `typed-css-modules-loader`](https://github.com/Megaputer/dts-css-modules-loader/blob/9bcb50bafb8d7cc42ddd5614f8e4b9b22c0dd7ff/package.json#L2)

See package here: https://www.npmjs.com/package/typed-css-modules-loader

Is it possible to use this module at the moment?

**Comment by c0gnize** [comment]
Thank you for your comment. I changed package name by commit e0df47c. Firstly this loader was named as `typed-css-modules-loader` but then I found the name is already used.

As far as I know, [typed-css-modules-loader](https://www.npmjs.com/package/typed-css-modules-loader) must be placed in a separate rule because uses [typed-css-modules](https://github.com/Quramy/typed-css-modules) which can process only css files.

`dts-css-modules-loader` just uses output of `css-loader`.
